component/prometheus/exporter/blackbox/blackbox.go

```diff
@@ -41,6 +41,9 @@ func buildBlackboxTargets(baseTarget discovery.Target, args component.Arguments)
 		for k, v := range baseTarget {
 			target[k] = v
 		}
+		for k, v := range tgt.ExtraLabels {
+			target[k] = v
+		}
```

Comment:
Should these be allowed to override existing labels?

Reply:
Good point, moved this to set first so that any subsequently set labels will override what's set in `extra_labels`.